### DISCUSSION TITLE
Captured 37kHz Panasonic code is transmitted but has no effect on TV

### DISCUSSION THREAD

**Discussion by sja1440** [bengtmartensson/IrScrutinizer#425] [Q&A] [answered]
I am running IrScrutinizer AppImage (v2.3.0) with an Arduino Nano with GirsLite v1.0.3 firmware. My PC is running an updated Ubuntu 20.04.

Using this system I can successfully capture codes from several devices.  These devices, with one exception, then respond correctly when I transmit the codes using IrScrutinizer.

My problem is that I can capture the codes from my Panasonic TV (a TX-55HZ1000), but when the codes are transmitted the TV does not react.

I have verified that the codes are indeed transmitted by using a second PC running IrScrutinizer with an identically configured Nano - the codes are captured correctly by this second device.

Can somebody help me understand what is going wrong please?

**Comment by bengtmartensson** [comment]
First thing to try is always to send, say, 3 copies of the signal (Sending hw -> Count (slightly below the middle)).

What, in detail, are the signals that you are (unsuccessfully) trying?

**Reply by sja1440** [reply]
Many thanks for your quick suggestion. Alas it did not work. 

The code that I am using for the test is the key '1'. Though the problem occurs with most if not all of the TV's codes.  

This is what I captured:
Decode = Panasonic: {D=128,F=16,S=0}, beg=0, end=99, reps=1
IRP = {37.0k,437,msb}<1,-1|1,-3>(8,-3,A:48,1,-65m)3{A=0x400401000809}
Freq=37037Hz[][+3495,-1398,+437,-437,+437,-1398,+437,-437,+437,-437,+437,-437,+437,-437,+437,-437,+437,-437,+437,-437,+437,-437,+437,-437,+437,-437,+437,-437,+437,-1398,+437,-437,+437,-437,+437,-437,+437,-437,+437,-437,+437,-437,+437,-437,+437,-437,+437,-437,+437,-1398,+437,-437,+437,-437,+437,-437,+437,-437,+437,-437,+437,-437,+437,-437,+437,-437,+437,-437,+437,-437,+437,-437,+437,-437,+437,-1398,+437,-437,+437,-437,+437,-437,+437,-437,+437,-437,+437,-437,+437,-437,+437,-1398,+437,-437,+437,-437,+437,-1398,+437,-65535][]

Transmitting the Scrutinized code does not work.

So I thought I'd try inserting the decode (Panasonic: {D=128,F=16,S=0}) into the Generate tab. That worked when I hit the Generate->Transmit button!

I then hit the Generate->To "Scrutinize signal" button and went back to the Scrutinize tab. This is what I see: 

Decode = Panasonic: {D=128,F=16,S=0}, beg=0, end=99, reps=1
IRP = {37.0k,432,msb}<1,-1|1,-3>(8,-3,A:48,1,-74m)*{A=0x400401000809}
Freq=37000Hz[][+3456,-1728,+432,-432,+432,-1296,+432,-432,+432,-432,+432,-432,+432,-432,+432,-432,+432,-432,+432,-432,+432,-432,+432,-432,+432,-432,+432,-432,+432,-1296,+432,-432,+432,-432,+432,-432,+432,-432,+432,-432,+432,-432,+432,-432,+432,-432,+432,-432,+432,-1296,+432,-432,+432,-432,+432,-432,+432,-432,+432,-432,+432,-432,+432,-432,+432,-432,+432,-432,+432,-432,+432,-432,+432,-432,+432,-1296,+432,-432,+432,-432,+432,-432,+432,-432,+432,-432,+432,-432,+432,-432,+432,-1296,+432,-432,+432,-432,+432,-1296,+432,-74736][]
The Decodes are identical, but the IRP and raw are different. Scrutinize->Transmit now produces the expected result on the TV.

Do I have a hardware issue? Bad clock?  Or does this happen occasionally with any hardware?

**Comment by bengtmartensson** [comment, marked as answer]
So if you capture them as "parametric", it should work?!

**Comment by sja1440** [comment]
Thanks!  After re-reading the manual, I think I now understand.  The Scrutinize window Transmit button sends the captured code exactly as received (ie the contents of the data window).  I'd assumed by mistake that it transmitted the "cleaned" version of the code.

Every thing is now clear!  Many thanks for this incredibly useful project.
